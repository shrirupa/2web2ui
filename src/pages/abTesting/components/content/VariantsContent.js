import React from 'react';

import { PageLink } from 'src/components';

const VariantsContent = () => (
  <div>
    <p>The templates you've selected will appear here, along with their results once this test concludes.</p>
    <p>If you need to create a new template, <PageLink to='/templates'>head over to the templates page</PageLink>.</p>
  </div>
);

export default VariantsContent;
