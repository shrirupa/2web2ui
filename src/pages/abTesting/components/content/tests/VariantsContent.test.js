import { shallow } from 'enzyme';
import React from 'react';
import VariantsContent from '../VariantsContent';

describe('Variants Content Component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<VariantsContent />);
  });

  it('should render help content', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
