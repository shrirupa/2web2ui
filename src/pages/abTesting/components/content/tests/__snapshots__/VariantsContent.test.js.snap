// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Variants Content Component should render help content 1`] = `
<div>
  <p>
    The templates you've selected will appear here, along with their results once this test concludes.
  </p>
  <p>
    If you need to create a new template, 
    <PageLink
      to="/templates"
    >
      head over to the templates page
    </PageLink>
    .
  </p>
</div>
`;
